packages/apps/plugins/plugin-markdown/src/types.ts

```diff
@@ -17,17 +16,16 @@ export enum MarkdownAction {
   CREATE = `${MARKDOWN_ACTION}/create`,
 }
 
+// TODO(burdon): This is just pretending not to be an ECHO object.
```

Comment:
It's not an echo object because local files also implements it and it's not using echo. If we were using local-only spaces instead of deepsignal then maybe it actually would be an echo object.

Reply:
in that case it should not rely on ECHO type definitions!